app/assets/javascripts/maintainsettlorsfrontend.js

```diff
@@ -61,6 +61,15 @@ $(document).ready(function() {
     // Assign aria-labbledby to the dynamically created country input
     if ($(".autocomplete-wrapper .error-message").length) $(".autocomplete__wrapper #value").attr('aria-labelledby', 'error-message-input');
 
+    // Override autocomplete styles to apply correct error component design pattern
+    if ($(".autocomplete-wrapper .error-message").length) $(".autocomplete__wrapper input").addClass('form-control-error');
+    $('.autocomplete__wrapper input').focus(function(e){
+        if ($(".autocomplete-wrapper .error-message").length) $(".autocomplete__wrapper input").css({"border" : "4px solid #0b0c0c", "-webkit-box-shadow" : "none", "box-shadow" : "none"});
```

Comment:
@jassalrichy could this be done with adding a css class to the input rather than directly setting CSS?

Reply:
@adamconder tried adding a css class, however the class seems to get overwritten by the autocomplete JavaScript. This is a fix that gets around that unless the autocomplete service update their JS file to take account of the design pattern for country input error styles.